project/src/main/java/com/google/sps/utilities/CommonUtils.java

```diff
@@ -1,9 +1,21 @@
 package com.google.sps.utilities;
```

Comment:
Add test for the public functions fo the util class.

Reply:
Done